src/admin-views/attendees-email.php

```diff
@@ -40,6 +40,21 @@
 			<input type="text" name="email_to_address" id="email_to_address" value="">
 		</label>
 		<input type="hidden" value="email" name="action" />
+		<fieldset class="list-or-table__section">
+			<legend><?php esc_html_e( 'Send data as a List or Table?:', 'event-tickets' ); ?></legend>
```

Comment:
I think since we have a question mark, the colon isn't needed.
```suggestion
			<legend><?php esc_html_e( 'Send data as a List or Table?', 'event-tickets' ); ?></legend>
```